src/components/molecules/projectDisplay/RewardCard.tsx

```diff
@@ -18,6 +18,17 @@ export const RewardCard = ({
 }: Props) => {
   const { t } = useTranslation()
   const {project} = useProjectContext()
+  const rewardStockRemaining = reward.maxClaimable ? reward.maxClaimable - reward.sold : -1;
+
+  const renderRewardAvailability = () => {
+    if(rewardStockRemaining > 0) {
+      return <><span style={{color: '#DF3634'}}>{rewardStockRemaining + ` ${t('remaining')}`}</span> <span style={{fontSize: "10px", position: "relative", top: "-2px"}}>&#8226;</span> </>;
+    } else if (rewardStockRemaining === 0) {
+      return <><span style={{color: 'neutral.600', fontWeight: 700}}>{t('Sold Out')}</span> <span style={{fontSize: "10px", position: "relative", top: "-2px"}}>&#8226;</span> </>;
+    } else {
```

Comment:
I don't think the color='neutral.600' would apply to a element, use what's mentioned in the previous comment.